packages/create-cloudflare/e2e-tests/helpers.ts

```diff
@@ -116,7 +129,12 @@ export const testProjectDir = (suite: string) => {
 	const clean = (suffix: string) => {
 		const path = getPath(suffix);
 		if (existsSync(path)) {
-			rmSync(path, { recursive: true, force: true });
+			rmSync(path, {
+				recursive: true,
+				force: true,
+				maxRetries: 10,
+				retryDelay: 500,
```

Comment:
@jculvey are we ok with these? or shall I try to remove them?

Reply:
I'm inclined to say remove them so that we can get to the bottom of what this issue is. If we can't, and we're getting a lot of these failures, we can throw them back in.

Reply:
removed 🙂